dev/integration_tests/flutter_gallery/windows/runner/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ target_compile_definitions(${BINARY_NAME} PRIVATE "NOMINMAX")
 # Add dependency libraries and include directories. Add any application-specific
 # dependencies here.
 target_link_libraries(${BINARY_NAME} PRIVATE flutter flutter_wrapper_app)
+target_link_libraries(${BINARY_NAME} PRIVATE "dwmapi.lib")
```

Comment:
I'm assuming we've verified this is available on Win7?

Reply:
Yup. I tested this builds & runs on Windows 7.

Reply:
FYI the original PR was reverted because the integration test crashed if the machine didn't support dark mode. The feature itself worked as expected though.

Reply:
Awesome - now that you mention Vista I think I looked this up on the last PR and remember seeing it mentioned that it was introduced with Vista. Sounds good! Thanks.